server/models/Message.js

```diff
@@ -1,9 +1,9 @@
 const mongoose = require("mongoose");
-
+var findOrCreate = require('mongoose-findorcreate')
```

Comment:
Change `var` to `let`, or change to  `const` if not reassigned

Reply:
again... I don't know what this is.